Resources/Prototypes/_Mono/Shipyard/iapetus.yml

```
@@ -17,7 +17,7 @@
   class:
   - Science
   engine:
-  - Plasma
+  - Uranium
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)